torch/csrc/distributed/c10d/Utils.hpp

```diff
@@ -4,6 +4,7 @@
 #include <c10/util/Exception.h>
 #include <c10/util/accumulate.h>
 #include <c10/util/irange.h>
+#include <fmt/format.h>
```

Comment:
This change added the FMT header, but it's not included in torch nightly wheels or bundled with `python setup.py install`, and caused pytorch extension build to fail.

Reply:
cc @malfet @atalman